### PR TITLE
Postgres changelogs fix

### PR DESCRIPTION
**Pull request by AlexanderSashchenko**
The issue is https://datical.atlassian.net/browse/DAT-6814

### CODE REVIEW

File: src/main/resources/liquibase/harness/change/changelogs/postgresql/9.5/datatypes.network.xml [hunk 1]
```diff
@@ -5,10 +5,14 @@
...
          http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
     <changeSet id="1" author="r2">
-        <createTable tableName="datatypes12_test_table">
-            <column name="serial" type="serial"/>
-            <column name="bigserial" type="bigserial"/>
-            <column name="macaddr8" type="macaddr8" />
+        <createTable tableName="datatypes.network_test_table">
+
+		    <!-- NETWORK TYPES -->
+            <!-- https://www.postgresql.org/docs/12/datatype-net-types.html -->
```
Comment (KushnirykOleh) [inline src/main/resources/liquibase/harness/change/changelogs/postgresql/9.5/datatypes.network.xml:11]: plz change version to corresponding (9.5 in this case) or delete this comment

File: src/main/resources/liquibase/harness/change/changelogs/postgresql/9/datatypes.network.xml [hunk 2]
```diff
@@ -5,10 +5,14 @@
...
          http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
     <changeSet id="1" author="r2">
-        <createTable tableName="datatypes12_test_table">
-            <column name="serial" type="serial"/>
-            <column name="bigserial" type="bigserial"/>
-            <column name="macaddr8" type="macaddr8" />
+        <createTable tableName="datatypes.network_test_table">
+
+		    <!-- NETWORK TYPES -->
+            <!-- https://www.postgresql.org/docs/12/datatype-net-types.html -->
```
Comment (KushnirykOleh) [inline src/main/resources/liquibase/harness/change/changelogs/postgresql/9/datatypes.network.xml:11]: same as above. Change link to 9th postgre doc version or delete

File: src/main/resources/liquibase/harness/change/changelogs/postgresql/datatypes.geometric.xml [hunk 3]
```diff
@@ -13,12 +13,13 @@
             <column name="line" type="line" />
             <!--
```
Comment (KushnirykOleh) [inline src/main/resources/liquibase/harness/change/changelogs/postgresql/datatypes.geometric.xml:14]: obsolete, delete this comment

**Review by KushnirykOleh — CHANGES_REQUESTED** [review]
minor nits related to comments

File: src/main/resources/liquibase/harness/change/changelogs/postgresql/9.5/datatypes.network.xml (hunk 1 above)
Comment (AlexanderSashchenko) [reply src/main/resources/liquibase/harness/change/changelogs/postgresql/9.5/datatypes.network.xml:11]: +

File: src/main/resources/liquibase/harness/change/changelogs/postgresql/9/datatypes.network.xml (hunk 2 above)
Comment (AlexanderSashchenko) [reply src/main/resources/liquibase/harness/change/changelogs/postgresql/9/datatypes.network.xml:11]: +

File: src/main/resources/liquibase/harness/change/changelogs/postgresql/datatypes.geometric.xml (hunk 3 above)
Comment (AlexanderSashchenko) [reply src/main/resources/liquibase/harness/change/changelogs/postgresql/datatypes.geometric.xml:14]: +

**Review by KushnirykOleh — APPROVED** [review]

**Pull request merged**